src/mono/CMakeLists.txt

```diff
@@ -206,31 +210,15 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS" OR CMAKE_SYSTEM_NAME STREQUAL "tvOS")
   add_definitions(-D_THREAD_SAFE)
   set(HOST_DARWIN 1)
   set(HOST_IOS 1)
+  if(CMAKE_SYSTEM_NAME STREQUAL "tvOS")
+    set(HOST_TVOS 1)
+  endif()
   set(PTHREAD_POINTER_ID 1)
   set(USE_MACH_SEMA 1)
   set(DISABLE_EXECUTABLES 1)
   set(DISABLE_CRASH_REPORTING 1)
   set(ENABLE_MONOTOUCH 1)
   add_definitions(-DMONOTOUCH=1)
-  set(HAVE_SYSTEM 0)
-  if(CMAKE_SYSTEM_NAME STREQUAL "tvOS")
-    set(HOST_TVOS 1)
-  endif()
-
-  # Force some defines
-  set(HAVE_GETPWUID_R 0)
-  set(HAVE_SYS_USER_H 0)
```

Comment:
Why are these removed ? They were in the configure version probably because they are detected to be present, but don't work/shouldn't be used.